frontend/src/schemas/SchemaSelect.tsx

```diff
@@ -139,9 +139,9 @@ export default function SchemaSelect({ entry }: SchemaSelectProps) {
               </Typography>
             </Stack>
             <Stack sx={{ mt: 2 }} spacing={2} alignItems='center'>
-              <Accordion defaultExpanded sx={accordionStyling}>
+              <Accordion defaultExpanded sx={accordionStyling} slotProps={{ heading: { component: 'h2' } }}>
```

Comment:
I'm not sure we want this change. We specifically want the text inside the heading to be a `h3`. Consider reverting this change unless the upgrade guide specifically mentions this is the correct way to do accessibility.